easybuild/easyconfigs/p/pyGIMLi/pyGIMLi-20160803-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'pyGIMLi'
+version = '20160803'
+commit_id = 'a190c53'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pygimli.org/'
+description = """pyGIMLi is an open-source multi-method library for solving inverse
+ and forward tasks related to geophysical problems. Written in C++ and Python,
+ it offers both efficiency and flexibility allowing you to quickly build
+ your own robust inversion applications for the geophysical problem at hand."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/gimli-org/gimli/archive/']
+sources = ['%s.tar.gz' % commit_id]
+
+patches = ['%(name)s-%(version)s_openblas.patch']
+
+#pyGIMLi is picky about the version of CastXML/pygccxml/pyplusplus
+#Check scripts/buildThirdParty.sh
+builddependencies = [
+    ('CMake', '3.4.3'),
+    ('CastXML', '20160617'),
+    ('pygccxml', '20160706', '-Python-%(pyver)s'),
+    ('pyplusplus', '20160707', '-Python-%(pyver)s'),
+]
+
+dependencies = [
+    ('Python', '3.5.1'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s'),
+    ('Clang', '3.7.1'),
+    ('Boost', '1.60.0', '-Python-%(pyver)s'),
+    ('zlib', '1.2.8'),
+    ('Triangle', '1.6'),
+    ('CppUnit', '1.12.1'),
+    ('SuiteSparse', '4.5.3', '-METIS-5.1.0'),
+]
+
+separate_build_dir = True
+
+configopts = '-DBOOST_ROOT=$EBROOTBOOST '
+configopts += '-DBLAS_LIBRARIES=$EBROOTOPENBLAS/lib/libopenblas.%s ' % SHLIB_EXT
+configopts += '-DLAPACK_LIBRARIES=$EBROOTOPENBLAS/lib/libopenblas.%s ' % SHLIB_EXT
+configopts += '-DCASTXML_EXECUTABLE=$EBROOTCASTXML/bin/castxml '
+configopts += '-DPYGCCXML_PATH=$EBROOTPYGCCXML/lib/python%(pyshortver)s/site-packages/ '
+configopts += '-DPYPLUSPLUS_PATH=$EBROOTPYPLUSPLUS/lib/python%(pyshortver)s/site-packages/ '
+
+buildopts = 'gimli pygimli apps'
+
+preinstallopts = 'mkdir -p %(installdir)s/lib/python%(pyshortver)s/site-packages '
+preinstallopts += '&& cp -a ../gimli*/python/.  %(installdir)s/lib/python%(pyshortver)s/site-packages '
+preinstallopts += '&& cp -a ../easybuild_obj/lib/.  %(installdir)s/lib/ '
+preinstallopts += '&& mkdir %(installdir)s/bin '
+preinstallopts += '&& cp ../easybuild_obj/bin/*  %(installdir)s/bin ||'
```

Comment:
@hajgato I think doing it via `installopts` (so, *after* the `make install`) would be cleaner, no?

Also, doesn't this suggest there's a bug in the `Makefile`?